sample_xblocks/basic/problem.py

```diff
@@ -139,7 +155,7 @@ def student_view(self, context=None):
                         }
                     }
                     var handlerUrl = runtime.handlerUrl(element, 'check')
-                    $.post(handlerUrl, JSON.stringify(data)).success(handleCheckResults);
+                    $.post(handlerUrl, JSON.stringify(data)).success(handleCheckResults(data));
```

Comment:
you shouldn't put the data in here, you want to verify the data that is returned from the result, not the data you are sending correct? you should only pass in the function you wish to have called upon response.